Kickstarter.xcodeproj/project.pbxproj

```diff
@@ -2895,10 +2910,16 @@
 				A73924001D27230B004524C3 /* Kickstarter_Framework.framework in Frameworks */,
 				8A86D7EE24FDC84400037A7B /* GoogleAppMeasurement.framework in Frameworks */,
 				774D98DA23A9795100FC81C2 /* Optimizely.framework in Frameworks */,
+				4748C19125B7A2F80098E89E /* FirebaseCore.framework in Frameworks */,
 				D0B45B6B1EF858C00020A8DA /* KsApi.framework in Frameworks */,
 				D0D58D862257FAE000532AC1 /* Prelude.framework in Frameworks */,
 				D0D58D8E2257FAE000532AC1 /* Prelude_UIKit.framework in Frameworks */,
+				4748C14525B1021D0098E89E /* FirebaseABTesting.framework in Frameworks */,
```

Comment:
How did you determine the list of deps that was needed in the end?

Reply:
While setting it up, I was initially running into some compilation errors.  This [thread](https://github.com/firebase/firebase-ios-sdk/issues/9#issuecomment-387947163) from Firebase suggested I should take a look at the missing frameworks being referenced in the errors and manually link them.